firmware/include/widgets/clockWidget.h

```diff
@@ -4,6 +4,35 @@
 #include <globalTime.h>
 #include <widget.h>
 
+#ifndef CLOCK_FONT
+#define CLOCK_FONT DSEG7
+#endif
+
+#ifndef CLOCK_FONT_SIZE
+#define CLOCK_FONT_SIZE 200
+#endif
+
+// Some fonts are not really centered, use this to compensate
```

Comment:
I assume there is something in openfontrender or freetype that has metrics for this?

Reply:
Unfortunately, I could not find any other way to show the clock reliably without it.

And I tried a lot of things with getHeight(), calculateBoundingBox() and other stuff.

But feel free to give it a try. I would be **very** happy if we can avoid/reduce the manual compensations.